packages/blade/wdio.bstack.conf.ts

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable @typescript-eslint/prefer-nullish-coalescing */
+/* eslint-disable no-template-curly-in-string */
+import type { Options } from '@wdio/types';
+import { config as baseConfig } from './wdio.base.conf.js';
+
+const commonCapabilities = {
+  buildName: 'blade browserstack build',
+  buildIdentifier: '#${BUILD_NUMBER}',
+};
+
+const bstackConfig: Options.Testrunner = {
+  maxInstances: 5,
+  user: process.env.BROWSERSTACK_USERNAME,
+  key: process.env.BROWSERSTACK_ACCESS_KEY,
+
+  services: [
+    [
+      'browserstack',
+      {
+        buildIdentifier: '#${BUILD_NUMBER}',
+        browserstackLocal: true,
+      },
+    ],
+  ],
+  capabilities: [
+    {
+      browserName: 'Chrome',
+      'bstack:options': {
+        browserVersion: 'latest',
+        os: 'OS X',
+        osVersion: 'Sonoma',
+        ...commonCapabilities,
+      },
+    },
+    {
+      browserName: 'Firefox',
+      'bstack:options': {
+        browserVersion: 'latest',
+        os: 'OS X',
+        osVersion: 'Sonoma',
+        ...commonCapabilities,
+      },
+    },
+    // Skipping safari for now, our storybook is broken in safari desktop :(
```

Comment:
I am able to open it in my laptop 😕

Reply:
blade.razorpay.com? 
What's your safari version? I'm using v15.4

<img width="345" alt="image" src="https://github.com/razorpay/blade/assets/35374649/7514ce2a-98a0-447d-88b8-51475a4de596">